rotkehlchen/tests/api/test_nfts.py

```diff
@@ -185,6 +250,149 @@ def test_nft_balances_and_prices(rotkehlchen_api_server):
     assert FVal(price_usd) > 0
 
 
+@pytest.mark.parametrize('ethereum_accounts', [[TEST_ACC4, TEST_ACC5]])
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+@pytest.mark.parametrize('ethereum_modules', [['nfts']])
+def test_edit_delete_nft(rotkehlchen_api_server):
+    """Check that ignoring NFTs work as expected"""
+    db = rotkehlchen_api_server.rest_api.rotkehlchen.data.db
+    nft_map = {
+        TEST_ACC4: [
+            NFT(
+                token_identifier='_nft_0x57f1887a8bf19b14fc0df6fd9b2acc9af147ea85_26612040215479394739615825115912800930061094786769410446114278812336794170041',  # noqa: E501
+                background_color=None,
+                image_url='https://openseauserdata.com/files/3f7c0c7d1ba51e61fe05ef53875f9f7e.svg',  # noqa: E501
+                name='yabir.eth',
+                external_link='https://app.ens.domains/name/yabir.eth',
+                permalink='https://opensea.io/assets/ethereum/0x57f1887a8bf19b14fc0df6fd9b2acc9af147ea85/26612040215479394739615825115912800930061094786769410446114278812336794170041',  # noqa: E501
```

Comment:
Okay I won't do all of those here, but as you see something is wrong with the formatting in this file. Please fix it. 

```suggestion
permalink='https://opensea.io/assets/ethereum/0x57f1887a8bf19b14fc0df6fd9b2acc9af147ea85/26612040215479394739615825115912800930061094786769410446114278812336794170041',  # noqa: E501
```

Reply:
Hmm I think this could be just how github displays it. Could you check how it is displayed for you in the IDE?